src/cljs/time_tracker_web_nxt/views/timer.cljs

```diff
@@ -0,0 +1,194 @@
+(ns time-tracker-web-nxt.views.timer
+  (:require
+   [re-frame.core :as rf]
+   [reagent.core :as reagent]
+   [time-tracker-web-nxt.views.common :as common]
+   [time-tracker-web-nxt.utils :as utils]))
+
+(defn- duration-handler
+  [data key]
+  (fn [val]
+    (let [parsed (if (empty? val)
+                   0
+                   (js/parseInt val 10))]
+      (swap! data assoc key parsed))))
+
+(defn create-timer-widget []
+  (let [data (reagent/atom {:notes      ""
+                            :elapsed-hh 0
+                            :elapsed-mm 0
+                            :elapsed-ss 0})]
+    (fn []
+      (let [show?            (rf/subscribe [:show-create-timer-widget?])
+            clients          @(rf/subscribe [:clients])
+            selected-client  @(rf/subscribe [:selected-client])
+            projects         @(rf/subscribe [:projects])
```

Comment:
This is the intention of the question I posed earlier. If `clients` is the list of all clients in db, then I'd expect `projects` and `tasks` to mean the same. Why not make the notion of projects for a client more explicit?

Reply:
makes sense, making this change.